README.md

```diff
@@ -17,8 +17,123 @@ value from data whilst ensuring it is properly governed.
+omagserverName is the name of OMAG server that is hosting the OMAS that this integration daemon is partnered with. These fields contain the information necessary
+to connect to the Egeria Server which is hosting the Data Manager OMAS.
+
+```shell
+
+--data-raw '{
+    "class": "IntegrationServiceRequestBody",
+    "omagserverPlatformRootURL": "https://platformName:9443",
+    "omagserverName" : "omagServer",
+    
+  ````
+In the next section we see how to configure the name of this particular connector instance and the userID that will be used to connect to the 
+egeria OMAS server.
+
+```shell
+    "integrationConnectorConfigs" : [ 
+        {
+             "class": "IntegrationConnectorConfig",
+             "connectorName" : "connectorInstanceName",             
+             "connectorUserId" : "egeriaUserID",
+  ````   
+In the next section we declare a connection object that will provide the details necessary to make a connection to the postgres database.
+We also provide the fully qualified Java classname for the postgres connectors provider.
+The `recognizedConfigurationProperties` and the `configurationProperties`sections provides the names of any parameters we wish to pass to the postgres JDBC client.
+In this example we just pass the jdbc connection url and set ssl to false. The full list of possible parameters are detailed in the postgres documentation 
+[here.](https://jdbc.postgresql.org/documentation/head/connect.html)
+```shell
+  "connection" : 
+             { 
+                 "class" : "Connection",
+                 "userId" : "postgresUserID",
+                 "clearPassword":"postgresPassword",
+                 "connectorType" : 
+                 {
+                     "class" : "ConnectorType",
+                     "connectorProviderClassName" : "org.odpi.openmetadata.adapters.connectors.integration.postgres.PostgresDatabaseProvider"
+                     
+                 },
+                  "recognizedConfigurationProperties": [
+                            "url",
+                            "ssl"
+                        ],
+                 "configurationProperties":
+                {
+                    "url": "jdbc:postgresql://localhost:5432/postgres",
+                    "ssl" : "false"
+                }
+             },
+```
+In the final section of the request body is where we set the final configuration properties.
+`metadataSourceQualifiedName` specifies the Egeria qualified name of the entity that represents the Software Server entity represents the postgres server.
+If you have already created and entity to represent the postgres server simply enter the qualified name here. If you haven't already created the server
+entity, one will be automatically created by the integration daemon.
+
+```shell
+
+             "metadataSourceQualifiedName" : "postgresHostServer",
+             "refreshTimeInterval" : "3456", 
```

Comment:
units?

Reply:
Just to check - is this really MINUTES? 3456 is a lot, nearly daily? But if that's intentional, fine

Reply:
it is minutes, but this is configurable and an example, so closing.